integration_test/Dialect/Calyx/loop.mlir

```diff
@@ -0,0 +1,32 @@
+// This test lowers an SCF construct through Calyx, FSM and (TODO)
+// to RTL.
+// RUN: circt-opt %s \
+// RUN:     --lower-scf-to-calyx -canonicalize \
+// RUN:     --calyx-remove-comb-groups --canonicalize \
+// RUN:     --calyx-go-insertion --canonicalize \
+// RUN:     --lower-calyx-to-fsm --canonicalize
+
+// This is the end of the road (for now) for Calyx in CIRCT.
+// The materialized FSM now needs to be outlined from within the
+// calyx module, and within the Calyx module it can be instantiated
+// as any other HW component. The FSM will then be lowered through
+// the existing FSM-to-HW flow.
+
+func.func @main() {
```

Comment:
Can you add some basic CHECK line? Maybe just CHECK for an FSM Op or something until the todo is done?